### DISCUSSION TITLE
Documentation on useQuery misleading

### DISCUSSION THREAD

**Discussion by skorphil** [TanStack/query#6537] [Ideas]
Documentation states that `queryFn` is required https://tanstack.com/query/latest/docs/react/reference/useQuery#:~:text=QueryFunctionContext)%20%3D%3E%20Promise%3CTData%3E-,Required,-%2C%20but%20only%20if

However, I use it without `queryFn` https://github.com/TanStack/query/discussions/6532#discussioncomment-7844431 and works fine

**Comment by TkDodo** [comment]
The docs say:

> Required, but only if no default query function has been defined

and that is correct. If you leave out the queryFn, but you haven't defined a default queryFn somewhere, your query will go into error state with the `Missing queryFn` error when the next fetch is attempted.
